arkouda-docker/README.md

```diff
@@ -0,0 +1,87 @@
+# arkouda-full-stack
+
+## Background
+
+The arkouda-full-stack image starts a one-locale arkouda_server and provides an ipython interface to Arkouda.
+
+## Building arkouda-full-stack
+
+```
+# set env variables
+ARKOUDA_BRANCH_NAME=2022.10.13
+ARKOUDA_DISTRO_NAME=v2022.10.13
+ARKOUDA_DOWNLOAD_URL=https://github.com/Bears-R-Us/arkouda/archive/refs/tags/v2022.10.13.zip
+
+docker build --build-arg ARKOUDA_DISTRO_NAME=$ARKOUDA_DISTRO_NAME \
+             --build-arg ARKOUDA_DOWNLOAD_URL=$ARKOUDA_DOWNLOAD_URL \
+             --build-arg ARKOUDA_BRANCH_NAME=$ARKOUDA_BRANCH_NAME \
+             -f arkouda-full-stack -t hokiegeek2/arkouda-full-stack:$ARKOUDA_DISTRO_NAME .
```

Comment:
would it be worthwhile to have a version which doesn't contain your username?

Reply:
@pierce314159 yeah, I have a question out to Bob about creating a Bears-R-Us Dockerhub repo for these images.